translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,2386 @@
+msgstr ""
+"\n"
+"\t\t  # Criar o ClusterRoleBinding para user1, user2, e group1 utilizando o "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin —clusterrole=cluster-"
+"admin —user=user1 —user=user2 —group=group1"
+
+#: pkg/kubectl/cmd/create_rolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a RoleBinding for user1, user2, and group1 using the admin "
+"ClusterRole\n"
+"\t\t  kubectl create rolebinding admin --clusterrole=admin --user=user1 --"
+"user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t  # Criar uma RoleBinding para user1, user2, e group1 utilizando o admin "
+"ClusterRole\n"
+"\t\t  kubectl create rolebinding admin —clusterrole=admin —user=user1 —"
+"user=user2 —group=group1"
+
+#: pkg/kubectl/cmd/create_configmap.go:44
+msgid ""
+"\n"
+"\t\t  # Create a new configmap named my-config based on folder bar\n"
+"\t\t  kubectl create configmap my-config --from-file=path/to/bar\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with specified keys instead "
+"of file basenames on disk\n"
+"\t\t  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1."
+"txt --from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with key1=config1 and "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config --from-literal=key1=config1 --from-"
+"literal=key2=config2"
+msgstr ""
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config based na pasta bar\n"
+"\t\t  kubectl create configmap my-config —from-file=path/to/bar\n"
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config com chaves "
+"especificas em vez do nome do arquivo em disco\n"
+"\t\t  kubectl create configmap my-config —from-file=key1=/path/to/bar/file1."
+"txt —from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config com key1=config1 e "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config —from-literal=key1=config1 —from-"
+"literal=key2=config2"
+
+#: pkg/kubectl/cmd/create_secret.go:135
+msgid ""
+"\n"
+"\t\t  # If you don't already have a .dockercfg file, you can create a "
+"dockercfg secret directly by using:\n"
+"\t\t  kubectl create secret docker-registry my-secret --docker-"
+"server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-"
+"password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL"
+msgstr ""
+"\n"
+"\t\t  # Se você ainda não tem o arquivo .dockercfg, você pode gerar "
+"diretamente o dockercfg secret utilizando o comando:\n"
+"\t\t  kubectl create secret docker-registry my-secret —docker-"
+"server=DOCKER_REGISTRY_SERVER —docker-username=DOCKER_USER —docker-"
+"password=DOCKER_PASSWORD —docker-email=DOCKER_EMAIL"
+
+#: pkg/kubectl/cmd/top_node.go:65
+msgid ""
+"\n"
+"\t\t  # Show metrics for all nodes\n"
+"\t\t  kubectl top node\n"
+"\n"
+"\t\t  # Show metrics for a given node\n"
+"\t\t  kubectl top node NODE_NAME"
+msgstr ""
+"\n"
+"\t\t  # Mostra as métricas para todos os nodos\n"
```

Comment:
Maybe we can leave `nodes` here or change to `nós`? Sometimes the term `node` is simply not translated at all.

Reply:
done, thanks for all